server/conn.go

```diff
@@ -838,7 +838,14 @@ func queryStrForLog(query string) string {
 	return query
 }
 
-func errStrForLog(err error) string {
+func errStrForLog(err error, enableRedactLog bool) string {
+	if enableRedactLog {
+		// currently, only ErrParse is considered when enableRedactLog because it may contain sensitive information like
+		// password or accesskey
+		if parser.ErrParse.Equal(err) {
+			return "fail to parse SQL and can't desensitize when enable log redaction"
```

Comment:
```suggestion
			return "fail to parse SQL and can't redact when enable log redaction"
```

Reply:
may I ask for more detailed explanation 😂